src/pages/Cart/CartInfoBox/CartInfoBox.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import InfoMapBox from './InfoMapBox';
+import './CartInfoBox.scss';
+
+export default function CartInfoBox({ infoBox }) {
+  const [sum, setSum] = useState(0);
+
+  const priceSum = () => {
+    let arr = [];
+    for (let i = 0; i < infoBox.length; i++) {
+      arr.push(infoBox[i].price);
+    }
+
+    let sum = arr.reduce((acc, cur) => acc + cur);
+
+    setSum(sum);
```

Comment:
```suggestion
 const totalSelectedPrice = infoBox
      .map(priceList => priceList.price)
      .reduce((prev, cur) => prev + cur);
    setSum(totalSelectedPrice);
```
* map으로 조금 더 간편하게 쓸 수 있습니다

Reply:
아 함수안에서 사용가능하군요. 변경해보겠습니다!